cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1574,6 +1574,15 @@
   </property>
 
   <property>
+    <name>cdap.ugi.cache.expiration.ms</name>
```

Comment:
We could, however it might be confusing as it exposes internals to the configuration. On a normal cluster, the expiration of TGT and delegation tokens are usually quite large (days), having the same merely means taking the minimal between them.